web/beacon-app/src/config/sentry.ts

```diff
@@ -0,0 +1,26 @@
+import * as Sentry from '@sentry/react';
+import { BrowserTracing } from '@sentry/tracing';
+
+const initSentry = () => {
+  const dsn = process.env.REACT_APP_SENTRY_DSN;
+  const environment = process.env.REACT_APP_SENTRY_ENVIRONMENT
+    ? process.env.REACT_APP_SENTRY_ENVIRONMENT
+    : process.env.NODE_ENV;
+
```

Comment:
we are going to avoid using `process.env` directly, what do you think if you define those environments in `application/config/appConfig` and consume them everywhere it's needed?

Reply:
I've made the update. I didn't see the config file before.